renovate.json

```diff
@@ -1,248 +1,40 @@
-    },
-    {
-      "customType": "regex",
-      "fileMatch": [
-        "hack/config"
-      ],
-      "matchStrings": [
-        "NODE_DRIVER_REG_VERSION=\"(?<currentValue>.*)\""
-      ],
-      "datasourceTemplate": "github-releases",
-      "depNameTemplate": "kubernetes-csi/node-driver-registrar"
-    },
-    {
-      "customType": "regex",
-      "fileMatch": [
-        "hack/config"
-      ],
-      "matchStrings": [
-        "LIVENESS_PROBE_VERSION=\"(?<currentValue>.*)\""
-      ],
-      "datasourceTemplate": "github-releases",
-      "depNameTemplate": "kubernetes-csi/livenessprobe"
-    },
-    {
-      "customType": "regex",
-      "fileMatch": [
-        "hack/config"
-      ],
-      "matchStrings": [
-        "CSI_SNAPSHOT_VERSION=\"(?<currentValue>.*)\""
-      ],
-      "datasourceTemplate": "github-releases",
-      "depNameTemplate": "kubernetes-csi/external-snapshotter"
-    },
-    {
-      "customType": "regex",
-      "fileMatch": [
-        "deploy/images.tmp.env"
-      ],
-      "matchStrings": [
-        "registry.k8s.io/sig-storage/csi-snapshotter@(?<currentDigest>.*)(\"?) # (?<currentValue>.*)"
-      ],
-      "datasourceTemplate": "docker",
-      "depNameTemplate": "registry.k8s.io/sig-storage/csi-snapshotter"
-    },
-    {
-      "customType": "regex",
-      "fileMatch": [
-        "deploy/images.tmp.env"
-      ],
-      "matchStrings": [
-        "registry.k8s.io/sig-storage/csi-node-driver-registrar@(?<currentDigest>.*)(\"?) # (?<currentValue>.*)"
-      ],
-      "datasourceTemplate": "docker",
-      "depNameTemplate": "registry.k8s.io/sig-storage/csi-node-driver-registrar"
-    },
-    {
-      "customType": "regex",
-      "fileMatch": [
-        "hack/config"
-      ],
-      "matchStrings": [
-        "CSI_SIG_STORAGE_PROVISIONER_VERSION=\"(?<currentValue>.*)\""
-      ],
-      "datasourceTemplate": "github-releases",
-      "depNameTemplate": "kubernetes-csi/external-provisioner"
-    }
+  "prConcurrentLimit": 3,
+  "lockFileMaintenance": {
+    "enabled": false
+  },
+  "postUpdateOptions": [
+    "gomodTidy",
+    "gomodVendor"
+  ],
+  "labels": [
+    "release-note-none"
+  ],
+  "extends": [
+    ":gitSignOff"
   ],
   "packageRules": [
     {
-      "description": "Components",
-      "matchPackageNames": [
-        "kubevirt/kubevirt",
-        "kubevirt/containerized-data-importer",
-        "kubevirt/cluster-network-addons-operator",
-        "kubevirt/ssp-operator",
-        "kubevirt/hostpath-provisioner-operator",
-        "kubevirt/hostpath-provisioner",
-        "kubevirt/managed-tenant-quota",
-        "kubevirt-ui/kubevirt-plugin",
-        "kubevirt-ui/kubevirt-apiserver-proxy",
-        "kubernetes-csi/livenessprobe",
-        "kubernetes-csi/external-provisioner"
-      ],
-      "versioning": "regex:^v(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)(-(?<prerelease>.*))?$",
-      "dependencyDashboardApproval": true,
```

Comment:
I see this was true only for  "description": "Components", why this is no longer needed?

Reply:
What ever was in this file in the past is not relevant. This is a new use for renovate and the whole structure is new